config/settings/base.py

```diff
@@ -621,8 +642,11 @@
 # }
 # Any time estimate that is not defined here, will use the "review time" from the closest value defined
 
-SPRINT_HOURS_RESERVED_FOR_REVIEW = json.loads(env.str("SPRINT_HOURS_RESERVED_FOR_REVIEW",
-                                                      '{"null": 2, "1.9": 0.5, "2": 1, "3": 2, "5": 3, "5.1": 5}'))
+
+SPRINT_HOURS_RESERVED_FOR_REVIEW = preprocess_sprints_hours_reserved_for_review(json.loads(
+    env.str("SPRINT_HOURS_RESERVED_FOR_REVIEW", '{"null": 2, "1.9": 0.5, "2": 1, "3": 2, "5": 3, "5.1": 5}')
+))
```

Comment:
Nit: wouldn't using [object_hook](https://docs.python.org/3/library/json.html) be clearer than adding a new level of the function calls?

Reply:
yes, didn't know about it, it will make it more clean